addons/hr_recruitment/static/src/js/tours/hr_recruitment.js

```diff
@@ -6,36 +6,97 @@ var tour = require('web_tour.tour');
     edition: 'community'
 }, {
     trigger: '.o_app[data-menu-xmlid="hr_recruitment.menu_hr_recruitment_root"]',
-    content: _t('Want to <b>start recruiting</b> like a pro? <i>Start here.</i>'),
+    content: _t("Let's have a look at how to <b>improve</b> your <b>hiring process</b>."),
     position: 'bottom',
     edition: 'enterprise'
 }, {
     trigger: ".o-kanban-button-new",
-    extra_trigger: '.o_hr_recruitment_kanban',
-    content: _t("Click here to create a new job position."),
-    position: "bottom"
+    content: _t("Create your first Job Position."),
+    position: "bottom",
+    width: 195
+}, {
+    trigger: ".o_job_name",
+    extra_trigger: '.o_hr_job_simple_form',
+    content: _t("What do you want to recruit today? Choose a job title..."),
+    position: "right"
+}, {
+    trigger: ".o_job_alias",
+    extra_trigger: '.o_hr_job_simple_form',
+    content: _t("Choose an application email."),
+    position: "right"
+}, {
+    trigger: '.o_create_job',
+    content: _t('Let\'s create the position. An email will be setup for applications, and a public job description, if you use the Website app.'),
+    position: 'bottom',
+    run: function (actions) {
+        actions.auto('.modal:visible .btn.btn-primary');
+    },
 }, {
     trigger: ".oe_kanban_action_button",
     extra_trigger: '.o_hr_recruitment_kanban',
-    content: _t("Let\'s have a look at the <b>applications pipeline</b> for this job position."),
+    content: _t("Let\'s have a look at the applications pipeline."),
+    position: "bottom"
+}, {
+    trigger: ".o_copy_paste_email",
+    content: _t("Copy this email address, to paste it in your email composer, to apply."),
     position: "bottom"
 }, {
     trigger: ".breadcrumb-item:not(.active):last",
     extra_trigger: '.o_kanban_applicant',
-    content: _t("Use the breadcrumbs to <b>go back to the dashboard</b>."),
+    content: _t("Let’s go back to the dashboard."),
     position: "bottom"
 }, {
-    trigger: ".o_job_alias",
+    trigger: ".oe_kanban_action_button",
     extra_trigger: '.o_hr_recruitment_kanban',
-    content: _t("Try to send an email to this address, it will create an application automatically."),
+    content: _t("<b>Did you apply by sending an email?</b> Check incoming applications."),
     position: "bottom"
+}, {
+    trigger: ".oe_kanban_card",
+    extra_trigger: '.o_kanban_applicant',
+    content: _t("<b>Drag this card</b>, to qualify him for a first interview."),
+    position: "bottom",
+    run: "drag_and_drop .o_kanban_group:eq(1) ",
+}, {
+    trigger: ".oe_kanban_card",
+    extra_trigger: '.o_kanban_applicant',
+    content: _t("<b>Click to view</b> the application."),
+    position: "bottom"
+}, {
+    trigger: ".o_Chatter .o_ChatterTopbar_buttonSendMessage",
+    extra_trigger: '.o_applicant_form',
+    content: _t("<div><b>Try to send an email</b> to the applicant.</div><div><i>Tips: All emails sent or received are saved in the history here</i>"),
+    position: "bottom"
```

Comment:
I believe the issue is that you are showing the marker "too quickly", but I cannot test because the app doesn't even install on runbot.

From what I understand from your video, the marker is currently positioned on the "send message" button where it would be from the right-border of the screen if the chatter was positioned on the right, except that due to the screen size being too small, the chatter, that is first rendered as-if on the right, is then moved to the bottom, where it takes the full width of the screen, which makes "send message" move but the marker is already set in the middle.

Related file is `mail_enterprise/static/src/widgets/form_renderer/form_renderer.js`. The file is full of hacks that we can't get rid of until the form view is converted to OWL. And `mail/static/src/widgets/form_renderer/form_renderer.js`.

As a first step, I would advise to test on community first, where the chatter is always at the bottom, and see how that works there. Then depending on the result, we'll see what to do.

Reply:
I just tested on the Community version and I have the same bug.

There is a runbot with the right version of the code if you want to test.

Reply:
Ok thanks for checking it, I'll have a look on runbot.